test/test_psutil.py

```diff
@@ -2475,6 +2475,25 @@ def test_Popen(self):
             proc.wait()
             self.assertIsNotNone(proc.returncode)
 
+    @unittest.skipUnless(hasattr(psutil.Process, "environ"),
+                         "environ not available")
+    def test_environ(self):
+        self.maxDiff = None
```

Comment:
OSX has weird variables that might be missing in either Process.environ or os.environ.  I could not reproduce this on my local system but the context was truncated.  maxDiff = None makes the errors much easier to spot.